tutorial/practicals/LFRic/distributed_memory/2_annexed_dofs/psyclone.cfg

```diff
@@ -34,7 +34,7 @@
 # Author: A. R. Porter, STFC Daresbury Laboratory
 # Modified by: R. W. Ford, STFC Daresbury Laboratory
 #              J. Henrichs, Bureau of Meteorology,
-#              I. Kavcic, Met Office
+#              I. Kavcic and O. Brunt, Met Office
 
 # This is the PSyclone configuration file.
```

Comment:
Not yours, but in the light of the discussions in the ticket, please could you update this comment to explain why we have this separate file.